views/archivePreview.ejs

```diff
@@ -6,6 +6,16 @@
     <meta http-equiv="content-type" content="text/html;charset=UTF-8">
     <meta http-equiv="X-UA-Compatible" content="chrome=1">
     <link rel="stylesheet" href="/css/room.opentok.css" />
+    <script>
```

Comment:
Would you please make the inclusion of this script conditional based on a configuration on a configuration variable.

Reply:
looks good